src/mono/netcore/System.Private.CoreLib/src/System/Threading/EventWaitHandle.Unix.Mono.cs

```diff
@@ -2,98 +2,82 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Microsoft.Win32.SafeHandles;
-using System.Runtime.CompilerServices;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
 
 namespace System.Threading
 {
     public partial class EventWaitHandle
```

Comment:
I do not see anything Mono-specific in this file. Can it be in the shared partition and not have .Mono in name?